tests/test_evaluators.py

```diff
@@ -464,7 +464,7 @@ def setUp(self):
         self.counter = 1337
         self.evaluator = DummyEvaluator(counter=self.counter, filtered=True)
         self.triples_factory = Nations().training
-        self.model = DummyModel(triples_factory=self.triples_factory, automatic_memory_optimization=False)
+        self.model = DummyModel(triples_factory=self.triples_factory)
```

Comment:
should set `automatic_memory_optimization=False` in the dummy evaluator above

Reply:
Done in 9a6a26f0086974c9aeb924a4d0187d424664d23b.